transformer_engine/common/include/transformer_engine/multi_tensor.h

```diff
@@ -17,13 +17,58 @@
 extern "C" {
 #endif
 
+/*!  \brief Computes L2 norm for a list of contiguous tensors, providing the option to
+ *          immediately exit the kernel based on the value of the 'noop' tensor.
+ *
+ * \warning   This API is **experimental** and subject to change.
+ * \warning   Argument device_id is deprecated and will be removed in a future release.
+ *
+ *  \param[in]     chunk_size              Number of tensors per kernel launch,
+                                           limited by total size of kernel args.
+ *  \param[in]     noop_flag               Noop tensor.
+ *  \param[in]     tensor_lists            2D array of contiguous input tensors.
```

Comment:
We don't support non-contiguous tensors. Also, I don't think we require the `NVTETensor`s to be contiguous in memory.
```suggestion
 *  \param[in]     tensor_lists            2D array of input tensors.
```